build.env

```diff
@@ -54,11 +54,11 @@ ROOK_VERSION=v1.12.1
 ROOK_CEPH_CLUSTER_IMAGE=quay.io/ceph/ceph:v18
 
 # CSI sidecar version
-CSI_ATTACHER_VERSION=v4.4.3
-CSI_SNAPSHOTTER_VERSION=v6.3.3
-CSI_RESIZER_VERSION=v1.9.3
-CSI_PROVISIONER_VERSION=v3.6.3
-CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.9.3
+CSI_ATTACHER_VERSION=v4.5.0
+CSI_SNAPSHOTTER_VERSION=v7.0.0
+CSI_RESIZER_VERSION=v1.10.0
+CSI_PROVISIONER_VERSION=v4.0.0
```

Comment:
can you please check if any new RBAC changes are required for any new features as some of them are major releases?

Reply:
Sure

Reply:
> can you please check if any new RBAC changes are required for any new features as some of them are major releases?

Checked, there are no RBAC changes required.